packages/horizon-householder-appeal-publish/handler.js

```diff
@@ -31,6 +31,16 @@ function convertToSoapKVPair(key, value) {
     };
   }
 
+  if (value == null) {
```

Comment:
@MrSimonEmms not sure about this part as the only current null value was this one. 

<img width="519" alt="Capture d’écran 2021-03-03 à 13 17 00" src="https://user-images.githubusercontent.com/3695996/109804632-bf96de80-7c22-11eb-9678-aa1fef2a672f.png">

Reply:
worth a `value === null` to avoid any casting issues? 

idk, only thing i can see to comment on :)

Reply:
I wanted to also include undefined, is there a better way without adding a && ?

Reply:
```suggestion
  if (value)
```

But only works with all falsy